T012_M1_load_data.py

```diff
@@ -223,7 +230,7 @@ def student_failures_dictionary(filename: str) -> Dict[int or str, List[dict]]:
 
     keys = []  # List for they keys of the csv file
     # Seperate the keys with commas and remove the new line at the end
-    basic_import[0] = basic_import[0].strip('\n').split(sep=',')
+    basic_import[0] = basic_import[0].strip('\n ').split(sep=',')
     # For loop that loops through the first line of the csv file and appends the keys
     for i in basic_import[0]:
         keys.append(i)
```

Comment:
```suggestion
keys = basic_import[0][:]
```
loop unnecessary. basic_import[0] is a list of headers at this point (line 233). You do not need to iterate over that list adding one element at a time to create an identical list in a different variable name. (The second variable name isn't even necessary, but I understand wanting to do that for readability). Anyway, to create a clone of a list (referring to a separate object in memory - which is what you seem to want to do here) we were taught two different sytaxes:
    1) keys = basic_import[0].copy()
    2) keys = basic_import[0][:]